registries/adapters/dockerhub_adapter.go

```diff
@@ -222,24 +220,30 @@ func (r DockerHubAdapter) loadSpec(imageName string) (*bundle.Spec, error) {
 	if r.Config.Tag == "" {
 		r.Config.Tag = "latest"
 	}
-	req, err := http.NewRequest("GET", fmt.Sprintf(dockerHubManifestURL, imageName, r.Config.Tag), nil)
+
+	token, err := r.getBearerToken(imageName)
```

Comment:
It looks like the order of these calls has changed, is there a reason?

Reply:
Seems to make sense to me. Now all the request logic is grouped together. Better to grab the token and error early before constructing the request.

Reply:
@shawn-hurley yes, I changed the order on purpose. Before we would create the request, then go out and get the token, THEN use the request we created earlier. It seemed confusing to me. Felt better to get the token. If that failed, no need to create the request. 

Also, I wanted the request creation close to where it was called to illustrate that it has no bearing on the bearer token call.